compiler/linkname.go

```diff
@@ -140,7 +86,7 @@ func isMitigatedVarLinkname(sym SymName) bool {
 // on a function, where the function has a body, is known about.
 // These are unsupported "insert"-style go:linkname directives,
 // that we ignore as a link and handle case-by-case in native overrides.
-func isMitigatedInsertLinkname(sym SymName) bool {
+func isMitigatedInsertLinkname(sym symbol.Name) bool {
```

Comment:
The update to linkname.go in go1.20 to help with some of the new linkname types needed to be updated for the change in master where `SymName` became `symbol.Name`. This was the only merge conflict I fixed.